packages/ebsr/configure/src/main.jsx

```diff
@@ -26,12 +26,42 @@ export class Main extends React.Component {
     onConfigurationChanged: PropTypes.func
   };
 
+  onModelChanged = (model, key) => {
+    const { onModelChanged } = this.props;
+
+    switch (key) {
```

Comment:
No need for "switch" here, you only have one different case to treat.

Reply:
ok